### ISSUE TITLE
Easily add a monitor in any existing group in status page

### ISSUE THREAD

**Issue by simone80an** [louislam/uptime-kuma#2597]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

Actually if i want to add a new monitor in an existing group in status page i have to move the group at the top and then add the monitor, then move down again the group in the position i want.

### ✔️ Solution

I would like to add a monitor in any existing group in the status page without having to move groups.

**Comment by t34wrj** [comment]
Is this not a separate issue to the issue mentioned above? This issue is about adding a new monitor to a status page group that is, for example, at the bottom of the list. At the moment, it seems as though you have to move the group all the way to the top of the page in order to then add a monitor to that group....and so on.

**Comment by simone80an** [comment]
> Is this not a separate issue to the issue mentioned above? This issue is about adding a new monitor to a status page group that is, for example, at the bottom of the list. At the moment, it seems as though you have to move the group all the way to the top of the page in order to then add a monitor to that group....and so on.

Yes it's same issue. Hope resolved with the new major versione 2.x

**Comment by t34wrj** [comment]
Oh sorry - it looked different to me. It will be great to easily add new monitors to existing groups without having to drag and drop the groups all the way to the top of the page in order to add the new monitor to an existing status page group! Thank you.
